### ISSUE TITLE
Crashing after playing two maps on iw4x

### ISSUE THREAD

**Issue by whatcorey** [XLabsProject/iw4x-client#235]
My server disconnects and crashes every single time after two maps, i have no idea why. Can anyone help me determine the issue? It is a lan dedicated server
[console_mp.log](https://github.com/XLabsProject/iw4x-client/files/8608798/console_mp.log)

**Comment by mjkzy** [comment]
is there a crash dump provided? that would be more useful

**Comment by whatcorey** [comment]
I dont think so. I just get "server disconnected" after playing two maps and then the server closes. Its on LAN dedicated



Sent from my MetroPCS 4G LTE Android Device



-------- Original message --------
From: m ***@***.***>
Date: 5/3/22 2:39 AM (GMT-08:00)
To: XLabsProject/iw4x-client ***@***.***>
Cc: whatcorey ***@***.***>, Author ***@***.***>
Subject: Re: [XLabsProject/iw4x-client] Crashing after playing two maps on iw4x (Issue #235)


is there a crash dump provided? that would be more useful

**Comment by Dss0** [comment]
does the server actually crash or does it just stop hosting a match (console stays open), if the former happens open the "minidumps" folder in your dedicated server folder and upload any dumps that may be in there. The latter would simply be caused by a typo in the map rotation or a broken mod.

**Comment by whatcorey** [comment]
yes i have minidumps, it also it says "server disconnected" after the two maps play. Heres the minidump. It wouldnt let me upload the dump so i renamed it .txt, just rename it .dmp
[iw4x-v0.6.1-20220430173742.txt](https://github.com/XLabsProject/iw4x-client/files/8613954/iw4x-v0.6.1-20220430173742.txt)

**Comment by mjkzy** [comment]
why don't you get off iw4x v0.7 for starters?

**Comment by whatcorey** [comment]
Its the one i got off the website xlabs



Sent from my MetroPCS 4G LTE Android Device



-------- Original message --------
From: m ***@***.***>
Date: 5/3/22 1:59 PM (GMT-08:00)
To: XLabsProject/iw4x-client ***@***.***>
Cc: whatcorey ***@***.***>, Author ***@***.***>
Subject: Re: [XLabsProject/iw4x-client] Crashing after playing two maps on iw4x (Issue #235)


why don't you get off iw4x v0.7 for starters?

**Comment by Dss0** [comment]
latest version is 0.7.1, this is also the version distributed via our official site so please update your server.

**Comment by whatcorey** [comment]
> latest version is 0.7.1, this is also the version distributed via our official site so please update your server.
[iw4x-v0.7.1-20220503144636.txt](https://github.com/XLabsProject/iw4x-client/files/8615550/iw4x-v0.7.1-20220503144636.txt)

I updated, still crashing

**Comment by Dss0** [comment]
try closing razer cortex.

**Comment by diamante0018** [comment]
Closing this in 7 days if we don't hear back from you @whatcorey

**Comment by diamante0018** [comment]
Closing this as I got no reply from OP
